ansible_builder/main.py

```diff
@@ -210,6 +216,11 @@ def prepare_prepended_steps(self):
     def prepare_appended_steps(self):
         additional_append_steps = self.definition.get_additional_commands()
         if additional_append_steps:
+            if not isinstance(additional_append_steps, dict):
+                raise TypeError((
+                    MessageColors.FAIL + "Expected 'additional_build_steps' to be a dictionary with keys 'prepend' and/or 'append',\n"
+                    f"found a {type(additional_append_steps)} instead" + MessageColors.ENDC
+                ))
```

Comment:
I'm not sure this needs to be in here... prepend is always the first thing that runs, and if the syntax in the definition file is:
```
additional_build_steps: |
    - RUN echo Oh hey this is an additional command!
    - RUN ls -la /etc
``` 
then `prepare_prepended_steps()` will always catch this, I think...